bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation
+     */
+    PointType getLocation();
```

Comment:
If I understand your code correctly this method could also return `null`. You should add this information to the JavaDoc e.g. `@return location of the current installation or null if the location is not set`.